modules/ossm-threescale-istio-adapter-troubleshooting.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/threescale_adapter/threescale-adapter.adoc
+// * service_mesh/v2x/threescale_adapter/threescale-adapter.adoc
+
+[id="ossm-threescale-istio-adapter-troubleshooting_{context}"]
+= 3scale Istio adapter troubleshooting
+
+You might want to check whether the 3scale Istio adapter is working as expected. If this is not the case, you can troubleshoot from the list of scenarios.
+
+Perform the following steps to check that the adapter is working correctly:
+
+. Ensure the _3scale-adapter_ pod is running in the control plane namespace, that is <istio-system>.
+. Check that the _3scale-adapter_ pod has printed out information about itself booting up, such as its version.
+. When performing requests to the services protected by the 3scale adapter integration, always try requests that lack the right credentials and ensure they fail. Check out the 3scale adapter logs to gather additional information.
+
+If the steps above do not help, see the following list, which you can use to troubleshoot the 3scale Istio adapter.
```

Comment:
This feels repetitive, we already had a list of things to try. Why isn't it a single list?

Reply:
I'd say these are verification steps, which will pass if the adpater is installed/configured correctly while the following list troubleshoots common reasons why the adapter isn't work.

@unleashed Please take a look. Cheers

Reply:
Yes @dfennessy that's right.